compendium/LogServiceImpl/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ endif()
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/resources/manifest.json.in
 	       ${CMAKE_CURRENT_BINARY_DIR}/resources/manifest.json)
+if(MSVC)
+  add_compile_definitions(_SILENCE_STDEXT_ARR_ITERS_DEPRECATION_WARNING)
```

Comment:
a short explanation of why this is necessary would help in the future. eventually we should be able to remove this and whomever is in a position to do that should know what is required to remove this compile macro definition.

Reply:
comment added